doc/modules/model_evaluation.rst

```diff
@@ -1955,6 +1957,37 @@ function::
   >>> mean_squared_log_error(y_true, y_pred)
   0.044...
 
+.. _mean_absolute_percentage_error:
+
+Mean absolute percentage error
+------------------------------
+The :func: The `mean_absolute_percentage_error` (MAPE), also known as mean absolute
+percentage deviation (MAPD), is a measure of prediction accuracy of a forecasting
+method in statistics, for example in trend estimation, also used as a loss function
```

Comment:
remove refs to forecasting. scikit-learn is not about time series forecasting although it can used with some overheads. Just say it's for regression

Reply:
DONE